docs/token-contracts/fa12/2-fa12-ligo.md

```diff
@@ -53,10 +53,10 @@ To deploy your asset contract;
 1. Choose the "Deploy" action under the "Configure" pick-list.
 1. In the "Storage" field, you shall see the initial data for this Smart
    Contract. You must update two items:
-       * Replace `YOUR_TZ_ADDRESS` with the address you copied from your faucet
-     file earlier
-       * Replace *both* occurrences of `TOKENS` with the total number of tokens
-     you wish to issue.
+   _ Replace `YOUR_TZ_ADDRESS` with the address you copied from your faucet
+   file earlier
+   _ Replace _both_ occurrences of `TOKENS` with the total number of tokens
```

Comment:
same here - leading underscore is not a valid markup, looks broken when rendered

Reply:
markdown is text format, of course you have full control over it...anyway, I see this is currently broken on the site... I can fix it outside of this pr